client/cypress/integration/settings/organization_settings_spec.js

```diff
@@ -6,10 +6,40 @@ describe("Settings", () => {
 
   it("renders the page and takes a screenshot", () => {
     cy.getByTestId("OrganizationSettings").within(() => {
-      cy.getByTestId("DateFormatSelect").should("contain", "DD/MM/YY");
```

Comment:
removed this line since the value now depends on the current setting, which the next test can change. running both tests one after another will change this value.